# ir_ranges
#  Values detected by the 16 infrared Sensors
# -1 = invalid or sensor offline
int16 range_si0
int16 range_si1
int16 range_si2
int16 range_si3
int16 range_si4
int16 range_si5
int16 range_si6
int16 range_si7
int16 range_si8
int16 range_si9
int16 range_si10
int16 range_si11
int16 range_si12
int16 range_si13
int16 range_si14
int16 range_si15